hudi-client/hudi-client-common/src/main/java/org/apache/hudi/index/bloom/HoodieBloomIndex.java

```diff
@@ -212,7 +212,7 @@ protected List<Pair<String, BloomIndexFileInfo>> loadColumnRangesFromMetaIndex(
     // also obtain file ranges, if range pruning is enabled
     context.setJobStatus(this.getClass().getName(), "Load meta index key ranges for file slices: " + config.getTableName());
 
-    String keyField = hoodieTable.getMetaClient().getTableConfig().getRecordKeyFieldProp();
+    String keyField = HoodieRecord.HoodieMetadataField.RECORD_KEY_METADATA_FIELD.getFieldName();
```

Comment:
Do you think we need to throw exception if `config.getColumnsEnabledForColumnStatsIndex()` does not contain the record keys?